lib/stat_tracker.rb

```diff
@@ -1,10 +1,39 @@
 require 'csv'
+require_relative 'games'
+require_relative 'teams'
+
 class StatTracker 
   def self.from_csv(locations)
     new(locations)
   end
 
   def initialize(locations)
-    
+    @games = create_objects(locations[:games], Games, self)
+    @teams = create_objects(locations[:teams], Teams, self)
+    @game_teams = create_objects(locations[:game_teams], GameTeams, self)
+
   end
+
+  def create_objects(path, statistics, tracker)
+    data = CSV.parse(File.read(path), headers: true, header_converters: :symbol)
+    data.map { |row| statistics.new(row, tracker) }
+  end
+
+  def highest_total_score
+      total_score.max
+  end
+  
+  def lowest_total_score
+      total_score.min
+  end
+
+  def total_score 
+     @games.map do |game|
+      home_goals = game.home_goals
+      away_goals = game.away_goals
+      home_goals + away_goals
+    end
+  end
```

Comment:
You made great progress on this Melchor! I like the addition of a helper method here, it makes it look very clean.

Reply:
I agree. I like this format and think the group should follow suit for the other methods as well. Once we have all of these methods written I think it might make sense to pull out the helper methods (like total_score) into a module file.